app/views/RoomInfoView/index.js

```diff
@@ -259,6 +272,21 @@ class RoomInfoView extends React.Component {
 		return null;
 	}
 
+	renderMessageButton = () => {
+		const { roomUser } = this.state;
+		const { theme } = this.props;
+		return (
+			<Button
```

Comment:
I think it's better if you use some Component like `ListItem`.
<img width="494" alt="Captura de Tela 2020-01-13 às 10 24 20" src="https://user-images.githubusercontent.com/29778115/72259335-f6f59680-35ee-11ea-87d6-7318efda8e4e.png">

Reply:
Thanks for reviewing my code sir. I made the changes.